### PR TITLE
Student to Web Developer

### PR DESCRIPTION
**Pull request by LouisPryer**
### Blog title
From Student to Web Developer
### Blog description
First blog post, I thought I would write up about what I learnt while doing my placement here and compare with what was taught while at university, give some insight for potential students looking for their placement years.
#### Author
- [X] I have checked for grammatical errors
- [X] I have checked for spelling errors
- [X] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by HarleyRowland** [comment]
Taking a review

**Comment by storer** [comment]
Ed 2

File: _posts/2015-03-30-From-Student-to-Web-Developer.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Like those that came before me, I embarked on my placement year at Holiday Extras after my second year at university. I had some experience with web developing before I started, however back then, this was not my main focus, in fact I wasn’t even sure myself what I should specialise in, both during my final year and after graduating. I was even close to switching my focus entirely and looking into a different degree. All that said, after my year at Holiday Extras I know now what I would like to focus in, and that is web development.
```
Comment (HarleyRowland) [inline _posts/2015-03-30-From-Student-to-Web-Developer.md]: I think this should be "Like those who came before me,"

**Comment by storer** [comment]
Looks good :+1:

**Comment by storer** [comment]
Looks good :+1:

**Comment by HarleyRowland** [comment]
Looks good +1

**Pull request merged**